# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/streams/ReadableStreamDefaultReader.h
# Commit: 4ec8f7211b07
# Full Hash: 4ec8f7211b0705481e717e365724a2e16d8601bb
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-03-19 09:41:58
# Regressor Bug: 1750298
# File Overlap Count: 1
# Description:
#   Bug 1750298 - Use custom ReadRequest for FetchStreamReader. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137681
# ==============================================================================

diff -r 1820f90df625 -r 4ec8f7211b07 dom/streams/ReadableStreamDefaultReader.h
--- a/dom/streams/ReadableStreamDefaultReader.h	Fri Mar 18 17:23:13 2022 +0000
+++ b/dom/streams/ReadableStreamDefaultReader.h	Fri Mar 18 17:46:32 2022 +0000
@@ -30,12 +30,8 @@
   NS_DECL_CYCLE_COLLECTION_CLASS_INHERITED(Read_ReadRequest, ReadRequest)
 
   RefPtr<Promise> mPromise;
-  /* This allows Gecko Internals to create objects with null prototypes, to hide
-   * promise resolution from Object.prototype.then */
-  bool mForAuthorCode = true;
 
-  explicit Read_ReadRequest(Promise* aPromise, bool aForAuthorCode = true)
-      : mPromise(aPromise), mForAuthorCode(aForAuthorCode) {}
+  explicit Read_ReadRequest(Promise* aPromise) : mPromise(aPromise) {}
 
   void ChunkSteps(JSContext* aCx, JS::Handle<JS::Value> aChunk,
                   ErrorResult& aRv) override;
